test/server/data/regression/gh-599/testfile.js

```diff
@@ -0,0 +1,23 @@
+import createCallsiteRecord from '../../../../../lib/errors/get-callsite.js';
+
+fixture `Test`
+    .page `http://example.com`;
+
+var obj = {
+    method1: () => {
+        throw new Error('Error on line - ' + (createCallsiteRecord('method1').lineNum + 1));
+    },
+
+    method2: () => {
+        return obj;
+    }
+};
+
+test('test', async () => {
+    await obj
+        .method2()
+        .method1()
+        .method2();
+
+    const actual = true;
```

Comment:
Original problem was able to reproduce, when we were declaring and initializing a variable after a call chain.